### ISSUE TITLE
[Bug] CSS EDITOR Always opens in "ALL WEBSITES" mode when invoked from firefox private window

### ISSUE THREAD

**Issue by aranapatona** [darkreader/darkreader#8511]
# Bug Report
## Website Address
Any.
## Bug Description
CSS Editor is always opened in ALL WEBSITES mode when opened from a Firefox private window.

## Steps To Reproduce
- Open a Private Window
- Go to example.com.
- Click dark reader ICON.
- Click Toggle Current Site check mark.
- Open More tab.
- Click Only for button.
- Switch to Static mode.
- Click Edit icon under Static button.

## Expected Behavior
Opens the CSS EDITOR for that specific website to enter your custom CSS

## Actual Behavior
Opens the CSS EDITOR for ALL WEBSITES   (legend ALL WEBSITES APPEARS AT TOP) 

## Screenshots
Expected Behavior: Opens for single site
![Expected Behavior Image](https://user-images.githubusercontent.com/19786351/160024599-f5477bfc-2645-42f4-b4de-26d68f4bf93e.png)

Actual Behavior: Opens for all websites
![Actual Behavior image](https://user-images.githubusercontent.com/19786351/160022732-779d30b9-6e6e-4ee8-acd6-06d1df859f17.png)

## System Information
- OS: Windows 10 64 bit 1709 Build 16299.1067
- Browser: Firefox 98.0.1 (64-bit)
- Dark Reader Version: 4.9.47

## Additional Context
Only happens while in a Private window

**Comment by Gusted** [comment]
The static theme's customization is based of the theme that's being used: 
![image](https://user-images.githubusercontent.com/25481501/160101502-d40fef86-5925-4efe-85c0-804cff3b596f.png)

If you want to create specific static theme for a site, you've select the specific theme for the website in order to only modify the static theme for that site.

**Comment by aranapatona** [comment]
That doesnt explain why it behaves differently in private mode than in normal mode. Also I dont see that option box anywhere.

**Comment by Gusted** [comment]
> That doesnt explain why it behaves differently in private mode than in normal mode.

Yeah I don't have a explanation for that either, private mode should still be able to read local's storage and the current codebase doesn't have any conditional for when dark reader is used in private mode. 

> Also I dont see that option box anywhere.


    Click Dark Reader icon.
    Click Dev tools (in the bottom-right corner).
    Click Preview new design.

**Comment by fragande** [comment]
Still having this issue in Firefox 153 with Darkreader 4.9.129. There doesn't seem to be a way to open the single page CSS editor at all anymore; seems completely broken.
